autoware_launch/config/planning/scenario_planning/common/motion_velocity_smoother/motion_velocity_smoother.param.yaml

```diff
@@ -60,3 +60,8 @@
     over_stop_velocity_warn_thr: 1.389       # used to check if the optimization exceeds the input velocity on the stop point
 
     plan_from_ego_speed_on_manual_mode: true  # planning is done from ego velocity/acceleration on MANUAL mode. This should be true for smooth transition from MANUAL to AUTONOMOUS, but could be false for debugging.
+
+    # force acceleration
+    adjusted_max_acceleration: 2.0
+    adjusted_max_jerk: 5.0
+    adjusted_max_lateral_acceleration: 1.0
```

Comment:
Can you create a namespace as follows for the better readability?
These are used in other packages like obstacle_cruise_planner for example.
```
force_acceleration:
   adjusted_max_acc: 2.0
   adjusted_max_jerk: 5.0
   ...
```

Reply:
Thank you for the review. I modified it in commit [ebb0187](https://github.com/autowarefoundation/autoware_launch/pull/1010/commits/ebb0187117579afe67f456ab96805b677607832).
Used abbreviation based on your suggestion.